irohad/main/impl/pg_connection_init.cpp

```diff
@@ -282,7 +282,8 @@ CREATE TABLE IF NOT EXISTS account_has_grantable_permissions (
 CREATE TABLE IF NOT EXISTS position_by_hash (
     hash varchar,
     height bigint,
-    index bigint
+    index bigint,
+    PRIMARY KEY (hash)
```

Comment:
This might create btree index instead of hash. Probably it is better to use explicit `CREATE INDEX` as it is done below.

Reply:
it appears that
> Only B-tree currently supports unique indexes.

([source](https://www.postgresql.org/docs/9.5/sql-createindex.html))

so I added a hash index and a unique constraint separately